libs/membership/website/src/lib/subscribe/subscribe-container.tsx

```diff
@@ -19,13 +19,20 @@ import {StripeElement, StripePayment} from '@wepublish/payment/website'
 import {useEffect, useMemo, useState} from 'react'
 import {OptionalKeysOf} from 'type-fest'
 
+/**
+ * If you pass the "deactivateSubscriptionId" prop, this specific subscription will be canceled when
+ * a new subscription is purchased. The subscription id is passed to the api that handles the
+ * deactivation. This is used for trial subscriptions or to replace legacy subscriptions like
+ * Payrexx Subscription. Other use cases are possible.
+ */
 export type SubscribeContainerProps<
   T extends OptionalKeysOf<RegisterMutationVariables> = OptionalKeysOf<RegisterMutationVariables>
 > = BuilderContainerProps &
   Pick<BuilderSubscribeProps<T>, 'fields' | 'schema' | 'defaults' | 'extraMoneyOffset'> & {
     successURL: string
     failureURL: string
     filter?: (memberPlans: MemberPlan[]) => MemberPlan[]
+    deactivateSubscriptionId?: string
```

Comment:
See comment above what this deactivateSubscriptionId does. It's basically a api functionality and very useful in this use case for Bajour but is also used for other use cases at Hauptstadt.